doc/source/whatsnew/v1.0.1.rst

```diff
@@ -16,6 +16,8 @@ including other versions of pandas.
 Bug fixes
 ~~~~~~~~~
 
+- Bug in :meth:`AbstractHolidayCalendar.holidays` when no rules were defined (:issue:`31415`)
```

Comment:
Did this change between 0.25.X and 1.0.0? If not should target 1.1.0 instead of 1.0.1